### DISCUSSION TITLE
time-series, prediction with new data?

### DISCUSSION THREAD

**Discussion by KazRyding** [pycaret/pycaret#3885] [Q&A]
Hi, as an analogy, imagine that today i have say 10 years of daily temperatures, i train a model with a forecast horizon of 7 so i predict the temperature forecast for 1 week. 
So tomorrow comes, and i have a new temperature data point.
What do i input as data for a new 7 day forecast? 
_prediction = predict_model(loaded_model, X=data)_
Will data be just tomorrow's temperature? 
or do i need a much larger data set including tomorrows temperature for a good prediction?

(Note: Its just an analogy, i'm just trying to get my head my head around how to use a saved time_series model)

**Comment by francoponti** [comment]
Have you got an answer to this?

**Reply by KazRyding** [reply]
No, not yet

**Reply by rvdb7345** [reply]
Did anyone find the solution yet by any chance?

**Reply by celestinoxp** [reply]
@ngupta23
